scripts/validate-dashboards.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+# List of files to exclude
+excludes=( "values.schema.json" "home.json" )
+exculde_string=$(IFS="|"; echo "${excludes[*]}")
```

Comment:
If it's a string, it should probably be quoted
```suggestion
exculde_string="$(IFS="|"; echo "${excludes[*]}")"
```